sys/zuse.hoon

```diff
@@ -4188,15 +4201,15 @@
         =+  [r y]=(jc-mul g k)
         =/  s  (pro.n `@`(inv.n k) `@`(sum.n z (mul r prv)))
         =/  big-s  (gte (mul 2 s) ^n)
-        :*  v=(add 27 (mix (end 0 1 y) ?:(big-s 1 0)))
+        :*  v=(mix (end 0 1 y) ?:(big-s 1 0))
             r=r
             s=?.(big-s s (sub ^n s))
         ==
       ::
       ++  ecdsa-raw-recover                             ::  get pubkey from sig
         |=  [has=@uvI sig=[v=@ r=@ s=@]]
         ^-  pont
-        ?>  ?&((lte 27 v.sig) (lte v.sig 34))
+        ?>  (lte v.sig 7)
```

Comment:
@belisarius222 seems sane, right?
Do you remember why this checks for such a large range, even though line 4204 means `v` can only ever be `0` or `1` (previously `27` or `28`)? It's not like the logic ever uses anything but the last bit from `v` anyway.

Reply:
I don't recall.